### ISSUE TITLE
Screen turns off right away when entering the loop() after setup()

### ISSUE THREAD

**Issue by VeniceInventors** [doubledave/ttgo-ntp-clock#6]
For some reason I can't figure, in the first iteration of the loop() the counter starts at 750.
It's then incremented to 751, and then the screen gets turned off as intended by "if (screen_time >= 750)".

I tried setting the counter to 0 at the end of setup but that doesn't make any difference.

My best guess is that during reset there might be some ghost state in button 2 which registers as the button being pressed, triggering the callback, and thus setting the counter to 750, otherwise I can't imagine where the 750 comes from.

**Comment by doubledave** [comment]
I am struggling to get my IDE working perfectly and am having issues with the wrong screen resolution on the Lilygo T-Display I have, but beside that, the current version in the main branch is working correctly for me in regard to the buttons and the counter.  Mine starts with the counter at 0 and in about 35 seconds counts to 250 and turns the backlight off.

My thoughts are the possibility that on your device, btn2 has gotten stuck or is otherwise that gpio line is held high.  That may be making it change to 750 every loop, preventing it from turning on the backlight.  With btn2 released, pressing and releasing btn1 should trigger the counter to change to 0 to allow for ~35 seconds of screen on time until the counter reaches 750.

@VeniceInventors would you like assistance with creating debug code to read out the button statuses onto the serial terminal?  That could determine if it's a physical issue, or if the pin numbers are correct for the 2 buttons used.

Alternatively, perhaps you don't have the same button library I've been testing with.  I have Arduino IDE 2.2.1 and in the library, I searched for "button2" which allowed for installing "Button2 by Lennart Hennigs" 2.2.4

**Comment by VeniceInventors** [comment]
I'm still on IDE 1.8.19 as I can't upgrade this Win8.1 machine to Win10, and wasn't able to get IDE 2.x to start, even the 2.11 version which is supposed to work on Win8.1, so it is possible that the button2 library is different, and I also noticed that I'm using button2 v1.4.0 while the latest available is 2.2.4 so I'll try to update it to know if that was the issue.

I'll make a test sketch to check the initial state of the buttons, with and without button2, but they appear to be working fine, other than the counter starting at 750 as if the button registered as LOW the first time it checks the buttons. (The pin numbers are correct, here 0 and 35 are pulled up by button2, so LOW=pressed).

I've changed the code to increment the counter by 250 instead, and that way it does stay on for 20 sec or so before it reaches 750, of course that requires clicking up to 3 times to turn it off.
I've been playing with the code a bit and made it activate light sleep after turning the screen off, saving a few extra mA.

It also appears that my clock is running fast, around 30 minutes per day, but I don't think I can fix that without digging deep into the espressif low-level timing registers.
